src/styles/block/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  flex-basis: 1024px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: $back-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+
+  &__img {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.nav {
+  display: flex;
+
+  &__list {
```

Comment:
The 'justify-self' property is not applicable in this situation. It should be removed.